tests/test_modeling_common.py

```diff
@@ -4179,7 +4186,7 @@ def get_mean_reldiff(failcase, x, ref, atol, rtol):
                             can_output_attn = "output_attentions" in inspect.signature(model_sdpa.forward).parameters
                             if not (self.has_attentions and can_output_attn) and output_attentions:
                                 continue
-                            for batch_size in [1, 5]:
```

Comment:
If possible, it would be nice to leave the original values, or `[1, 7]` (so as to test batched and unbatched)